apps/settings/style/settings.css

```diff
@@ -918,13 +923,12 @@ ul li > label.range-icons.volume input {
   height: auto;
 }
 
-#feedback-happy {
-  float: left;
-}
+html[dir="ltr"] #feedback-happy { float: left; }
```

Comment:
I’ll add a reference to https://bugzilla.mozilla.org/show_bug.cgi?id=1122918 while I’m at it